ntbs-integration-tests/NotificationPages/OverviewPageTests.cs

```diff
@@ -136,5 +136,23 @@ public async Task OverviewPageShowsCreateNewNotificationForThisPatientButton_IfN
             var document = await GetDocumentForUrl(url);
             Assert.NotNull(document.QuerySelector("#new-linked-notification-button"));
         }
+        
+        [Fact]
+        public async Task OverviewPageShowsDenotificationDetails_ForDenotifiedRecord()
+        {
+            // Arrange
+            var url = GetCurrentPathForId(Utilities.DENOTIFIED_ID);
+            var document = await GetDocumentForUrl(url);
+            Assert.NotNull(document.QuerySelector("#overview-denotification-details"));

```

Comment:
Definitely feel free to ignore, but it might be nice to confirm the text is displayed - just a simple `.contains("x" , y.TextContent)` nothing fancy.